lib/internal/Magento/Framework/App/Config.php

```diff
@@ -132,6 +134,6 @@ public function get($configType, $path = '', $default = null)
             $result = $this->types[$configType]->get($path);
         }
 
-        return $result !== null ? $result : $default;
+        return $result !== null ? $result : ($default !== null ? $default : '');
```

Comment:
I think we can simplify it 
```suggestion
        return $result ?? $default ?? '';
```

Here is example:
https://3v4l.org/SY8Ys

Reply:
@ihor-sviziev I added unit test